redash/controllers.py

```diff
@@ -450,7 +450,7 @@ def delete(self, visualization_id):
 class QueryResultListAPI(BaseResource):
     @require_permission('execute_query')
     def post(self):
-        params = request.json
```

Comment:
Updated! Using `get_json(force=True)` is not much shorter than `json.loads(request.data)`. :)